src/components/RightPane/Map/LazyUCIMap.js

```diff
@@ -0,0 +1,12 @@
+import React, { Suspense } from 'react';
+
+const LazyUCIMap = () => {
```

Comment:
This should no longer be in the PR as of daae514, as it's been consolidated into `RightPaneRoot`. I think GitHub might have given you an outdated version of the source tree to review since my requesting of your review predated that commit.